server/base/src/main/java/org/apache/accumulo/server/ServiceEnvironmentImpl.java

```diff
@@ -78,6 +79,22 @@ public String get(String key) {
       }
     }
 
+    @Override
+    public Map<String,String> getWithPrefix(String prefix) {
+      if (Property.isValidPropertyPrefix(prefix)) {
+        Property propertyPrefix = Property.getPropertyByKey(prefix);
```

Comment:
This check for valid prefix property can be avoided, by checking the returned object from `Property.getPropertyByKey(prefix)`:

```suggestion
      Property propertyPrefix = Property.getPropertyByKey(prefix);
      if (propertyPrefix != null && propertyPrefix.getType() == PropertyType.PREFIX) {
```

Reply:
Fixed